src/gui/mzroll/settingsform.cpp

```diff
@@ -499,3 +525,15 @@ void SettingsForm::setStringValue(QString key, QString value) {
      settings->setValue(key,value);
 }
 
+void SettingsForm::_smoothnessChanged(int value)
+{
+    smoothnessValue->setText(QString::number(value));
+    recomputeEIC();
+}
+
+void SettingsForm::_asymmetryChanged(int value)
+{
+    double actualAsym = static_cast<double>(value) / 100.0;
```

Comment:
@saifulbkhan Why the conversion to actualAsym? You can set the range and step size of the slider to give you the actual number you want. Was there a reason for converting it yourself?

Reply:
Unfortunately, sliders can only have integers for values and steps. That is why this complication.

Reply:
Oh yes. My bad.